docs/setup/pick-right-solution.md

```diff
@@ -150,6 +152,7 @@ KCluster.io          |              | multi-support | multi-support | [docs](htt
 KUBE2GO.io          |              | multi-support | multi-support | [docs](https://kube2go.io) | Commercial
 Madcore.Ai           | Jenkins DSL  | Ubuntu | flannel     | [docs](https://madcore.ai)                        | Community ([@madcore-ai](https://github.com/madcore-ai))
 Platform9        |              | multi-support | multi-support | [docs](https://platform9.com/managed-kubernetes/) | Commercial
+GiantSwarm.io        |              | CoreOS | flannel     | [docs](https://docs.giantswarm.io/) | Commercial
```

Comment:
`flannel` -> `flannel and/or Calico`
(we use flannel only onprem and always together with Calico)